kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementContainmentControl.java

```diff
@@ -96,9 +96,9 @@ public boolean onMove(double dx,
                 final Point2D mouseRelativeLoc = new Point2D(mouseX - parentAbsLoc.getX(),
                                                              mouseY - parentAbsLoc.getY());
                 //Children contains m_ghost and others excluding m_shape. This therefore moves m_ghost within children.
-                getParent().getLayoutHandler().add(ghost.get(),
-                                                   getParent(),
-                                                   mouseRelativeLoc);
+//                getParent().getLayoutHandler().add(ghost.get(),
```

Comment:
@jeremyary 
When dragging a subprocess under a stage in the first column, the ghost of the subprocess moves to the second column immediately and pushes the stage in the second column to the third column. 
Removing this line could fix the behavior.